### PR TITLE
Cloud Guide MFA and TFA updates

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Added section about SSH with MFA to the Enable SSH keys topic
- Updated magento-cloud CLI reference to latest (1.35.0) for update auth commands
- Cleaned up instructions for enabling TFA on Cloud accounts
- Replaced 2FA with TFA
- Added Security and Compliance search tags to Fastly and authentication topics
- Edits and syntax fixes

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/before/before-workspace-ssh.html
-  https://devdocs.magento.com/cloud/cdn/cloud-fastly.html
-  https://devdocs.magento.com/cloud/cdn/fastly-vcl-whitelist.html
-  https://devdocs.magento.com/cloud/cdn/fastly-waf-service.md
-  https://devdocs.magento.com/cloud/project/project-enable-mfa-enforcement.html
-  https://devdocs.magento.com/cloud/project/user-admin.html

whatsnew
Updated the Magento Cloud CLI reference for v1.35.0.

### CODE REVIEW

File: src/_includes/cloud/enable-ssh.md [hunk 1]
```diff
@@ -72,6 +74,9 @@ To create an SSH key pair:
        IdentityFile ~/.ssh/id_rsa
    ```
 
+   {:.bs-callout-info}
+   You can specify multiple SSH keys by adding multiple IdentityFile entries to your configuration.
```
Comment (meker12) [inline src/_includes/cloud/enable-ssh.md:78]:
```suggestion
   You can specify multiple SSH keys by adding multiple `IdentityFile` entries to your configuration.
```

File: src/cloud/project/user-admin.md [hunk 2]
```diff
@@ -94,12 +94,14 @@ The following example changes the environment-level role that is assigned to a u
 magento-cloud user:role alice@example.com --level environment --environment development --role admin
 ```
 
-{:.bs-callout-info}
-To list the available `magento-cloud` CLI commands, use the `magento-cloud list` command.
+{:.bs-callout-tip}
+Use the following command to list the available `magento-cloud` CLI commands: `magento-cloud list`
```
Comment (keharper) [inline src/cloud/project/user-admin.md:98]:
```suggestion
The `magento-cloud list` command displays all the `magento-cloud` CLI commands.
```

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 3]
```diff
@@ -301,9 +300,22 @@ To authenticate and reload the SSH certificate:
...
+
+If the SSH key is missing or invalid, the SSH connection request returns a `Permission denied (publickey)` error.
+
+```terminal
+Hello user-test (UUID: abaacca12-5cd1-4b123-9096-411add578998), you successfully authenticated, but could not connect to service oh2wi6klp5ytk-mc-35985-integration-nnulm4a--mymagento (reason: service doesn't exist or you do not have access to it)
+oh2wi6klp5ytk-mc-35985-integration-nnulm4a--mymagento@ssh.eu-3.magento.cloud: Permission denied (publickey).
+```
+{:.no-copy}
+
+To fix the problem, you might need to add the ssh key to your current session, or update the SSH configuration file to load your SSH keys automatically. See [Add a public ssh key][add public ssh key] and [Unable to access projects without MFA][].
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:313]:
```suggestion
To fix the problem, you might need to add the SSH key to your current session, or update the SSH configuration file to load your SSH keys automatically. See [Add a public ssh key][add public ssh key] and [Unable to access projects without MFA][].
```

File: src/cloud/project/user-admin.md [hunk 4]
```diff
@@ -56,9 +56,9 @@ Available commands:
 The following examples use the CLI to add a user, configure roles, and modify project assignments and assigned user roles.
 
 {:.procedure}
-Add a user and assign roles:
+To add a user and assign roles:
 
-1. Add the user:
+1. Add the user using Magento Cloud CLI.
```
Comment (meker12) [inline src/cloud/project/user-admin.md:61]:
```suggestion
1. Use the CLI to add the user.
```

File: src/_includes/cloud/enable-ssh.md [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
+You must create an SSH key pair on every machine and workspace that requires access to {{site.data.var.ece}} project source code and environments. The SSH keys allow you to connect to GitHub to manage source code and to connect to cloud servers without having to constantly supply your username and password.
+
+You can add multiple SSH keys for each system or workspace that you use.
 
 The SSH keys require the following:
 
-*  Set up SSH keys as the [Magento file system owner]({{ site.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
-*  Create the keys using the email address used for the GitHub account.
+-  Set up SSH keys as the [Magento file system owner]({{ site.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+-  Create the keys using the email address used for the GitHub account.
```
Comment (meker12) [inline src/_includes/cloud/enable-ssh.md:8]:
```suggestion
-  Create the keys using the GitHub account email address.
```

File: src/_includes/cloud/enable-ssh.md [hunk 6]
```diff
@@ -112,11 +117,12 @@ After adding the SSH keys, test the SSH connection to GitHub:
...
-*  Using the [{{site.data.var.ece}} Web Interface](#add-key-web)
+-  Using the [{{site.data.var.ece}} CLI](#add-key-cli)
+-  Using the [{{site.data.var.ece}} Web Interface](#add-key-web)
 
 ### Add a key using the CLI {#add-key-cli}
 
+{:.procedure}
 To add an SSH key using the CLI:
 
 1. Open a terminal application on your local.
```
Comment (meker12) [inline src/_includes/cloud/enable-ssh.md:128]:
```suggestion
1. Open a terminal application on your local workstation.
```

File: src/cloud/project/project-enable-mfa-enforcement.md (hunk 3 above)
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:313]:
```suggestion
To fix the problem, you might need to add the SSH key to your current session, or update the SSH configuration file to load your SSH keys automatically. See [Add a public SSH key][add public ssh key] and [Unable to access projects without MFA][].
```

File: src/cloud/project/user-admin.md [hunk 7]
```diff
@@ -19,17 +19,17 @@ You can manage access to {{site.data.var.ece}} projects by adding users and assi
...
+| **Super user** | Project | Administrator access to all project settings and Cloud environments. Super users can change settings and perform tasks on any environment, including creating and restoring [snapshots][]. |
+| **Project reader** | Project | View access to all project environments. Users with this role cannot perform tasks on any environment. However, you can configure environment-level permissions for users with this role to permit write access to a specific environment. |
+| **Admin** | Environment | Change settings and perform tasks on an environment, including merging with the parent environment |
 | **Contributor** | Environment | Push code and branch the environment |
 | **Reader** | Environment | View-only access to an environment |
 
 ## Add user authentication requirements
 
-For added security, Magento is introducing a project-level MFA enforcement feature to improve security for SSH access to Cloud infrastructure. See [MFA enforcement for SSH].
+For added security, Magento provides a project-level MFA enforcement feature to improve security for SSH access to Cloud infrastructure. See [MFA enforcement for SSH].
```
Comment (meker12) [inline src/cloud/project/user-admin.md:30]:
```suggestion
For added security, Magento provides project-level MFA enforcement to require two-factor authentication for SSH access to {{ site.data.var.ece }} project source code and environments. See [MFA enforcement for SSH].
```

File: src/cloud/project/project-enable-mfa-enforcement.md [hunk 8]
```diff
@@ -301,9 +300,22 @@ To authenticate and reload the SSH certificate:
...
+To fix the problem, you might need to add the SSH key to your current session, or update the SSH configuration file to load your SSH keys automatically. See [Add a public ssh key][add public ssh key] and [Unable to access projects without MFA][].
+
 <!--Link references-->
 [add public ssh key]: {{ site.baseurl }}/cloud/before/before-workspace-ssh.html#ssh-add-to-account
 [Cloud account page]: https://accounts.magento.cloud/user
 [Create an API token]: {{ site.baseurl }}/cloud/project/user-admin.html#create-an-api-token
-[Enable 2FA]: {{ site.baseurl }}/cloud/project/user-admin.html#enable-2fa-for-cloud-accounts
+[Enable TFA]: {{ site.baseurl }}/cloud/project/user-admin.html#enable-tfa-for-cloud-accounts
 [Manage user access]: {{ site.baseurl }}/cloud/project/user-admin.html
+[Unable to access projects without MFA]: {{ site.baseurl }}/cloud/before/before-workspace-ssh.html#unable-to -access-projects-without-mfa
```
Comment (meker12) [inline src/cloud/project/project-enable-mfa-enforcement.md:321]:
```suggestion
[Unable to access projects without MFA]: {{ site.baseurl }}/cloud/before/before-workspace-ssh.html#unable-to-access-projects-without-mfa
```

**Comment by meker12** [comment]
running tests

**Pull request merged**